### ISSUE TITLE
[Tab]: VoiceOver does not announce number of tabs

### ISSUE THREAD

**Issue by janhassel** [carbon-design-system/carbon#5083]
## Environment

> Operating system
macOS 10.14.6

> Browser
Chrome 79

> Assistive technology used to verify
VoiceOver

## Detailed description

> What version of the Carbon Design System are you using?
carbon-components-react@7.9.1

> What did you expect to happen?
When tabbing through tabs, VoiceOver should announce the number tabs in this group, e.g. "[label], selected, tab, 4 of 4"

> What happened instead?
VoiceOver always announces the current tab as 1 of 1.

> What WCAG 2.1 checkpoint does the issue violate?

## Steps to reproduce the issue

1. Open http://react.carbondesignsystem.com/?path=/story/tabs--default
2. Click a tab
3. Activate VoiceOver
4. Tab between the tabs

## Additional information

![carbon-tab](https://user-images.githubusercontent.com/28265588/72614042-85d91a80-3931-11ea-9894-b0162bb12630.gif)
